### PR TITLE
mgmt ibu: add negative test suite

### PR DESCRIPTION
**Pull request by trewest**
Add negative test suite to contain MGMT IBU negative tests

### CODE REVIEW

File: tests/lca/imagebasedupgrade/mgmt/negative/internal/tsparams/negativevars.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package tsparams
```
Comment (achuzhoy) [inline tests/lca/imagebasedupgrade/mgmt/negative/internal/tsparams/negativevars.go:1]: let's rename to negative_vars.go to be consitent with other negative_ names

**Review by achuzhoy — APPROVED** [review]

**Comment by achuzhoy** [comment]
just one minor comment
/lgtm
